src/CoreBundle/private/library/classes/TCMSResourceCollection.class.php

```diff
@@ -297,7 +298,12 @@ protected function CreateCSSResourceCollectionFile($sFileMD5, $aCSS)
 
                             $minify = ServiceLocator::getParameter('chameleon_system_core.resources.enable_external_resource_collection_minify');
                             if ($minify) {
-                                $sSubString = CssMin::minify($sSubString);
+                                // TODO use a service here (see MinifyJsJshrinkService) - if there are any problems with customer css content with new lib?
```

Comment:
Absolutely. This should be replaceable. A service to call the minification in between should not be a big deal.

Reply:
Hm, the Js minification even uses an event listener mechanism: \TCMSResourceCollection::dispatchJSMinifyEvent()

Reply:
Added a service for now.